api_fhir/converters/claimConverter.py

```diff
@@ -46,6 +47,19 @@ def to_imis_obj(cls, fhir_claim, audit_user_id):
         cls.check_errors(errors)
         return imis_claim
 
+    @classmethod
+    def get_reference_obj_id(cls, imis_claim):
+        return imis_claim.code
+
+    @classmethod
+    def get_fhir_resource_type(cls):
+        return FHIRClaim
+
+    @classmethod
+    def get_imis_obj_by_fhir_reference(cls, reference, errors=None):
+        imis_claim_code = cls.get_resource_id_from_reference(reference)
+        return Claim.objects.filter(code=imis_claim_code).first()
```

Comment:
why u use first here ? It can be more than 1 row ?

Reply:
I used the .first() method because the .filter(...) returns the queryset. I decided to use the .filter() instead of .get() method in order to avoid throwing exception when the object is missing. But you are right it could be a little bit confusing, so I  will move it to util method and use the `try ... except` in order to avoid throwing exception.